Worker Form

```diff
@@ -0,0 +1,2 @@
+Please refer the form here, based on https://github.com/Help-the-helpers/Vision/blob/master/Worker
+English: https://forms.gle/6CZ2TUcoxMJmkqzC9
```

Comment:
Can we have a English<->Hindi dictionary? i.e. If an employer searches for Plumber, will it include results for Hindi spelling OR word for Plumber?

Reply:
We have this as an API by Google.
https://translate.google.co.in/#view=home&op=translate&sl=en&tl=hi

Moreover, from the query pov, yes - this functionality needs to be incorporated. (technically we could associate each role with a code - number. ofcourse there are other solutions - will open a tech issue for this)
https://github.com/Help-the-helpers/Tech-And-Integrations/issues/2

Reply:
Also - pls make sure - for jobs/roles we should aim for entire list from TeamLease - autocomplete will hide the size of the list... w/ ability to add more jobs/roles as and when needed (we essentially need custom labels)... we don't want to limit - in the backend - we may need to call out synonyms OR duplicates if 2 jobs/roles mean the same thing...

Reply:
Also - think of MVP - so cut scope if you need to - early to market

Reply:
>  we may need to call out synonyms OR duplicates if 2 jobs/roles mean the same thing

Feedback incorporated here - https://github.com/Help-the-helpers/Tech-And-Integrations/issues/2

Agreed, in initial stages, for MVP, a small set works, and set and working with it will evolve.